README.md

```diff
@@ -15,24 +15,16 @@ solutions up and going with as little hindrance as possible.
 
 > Xenial and above
 
-We use snap's for packaging and installation, as of snapd 2.11 the following
-needs to be run
+## pre-reqs
+
+Some spells require that LXD be available and ready:
```

Comment:
If you plan to use conjure-up to deploy onto LXD containers on your local machine (aka the 'localhost' cloud type), you will need to set up LXD beforehand, as this requires sudo. If you haven't done that, you can configure storage and networking with 'sudo dpkg....' and wake up the lxd daemon with 'lxc finger'.